protocol/contracts/libraries/LibIncentive.sol

```diff
@@ -68,105 +70,125 @@ library LibIncentive {
+                    return _scaleReward(beans, 2_047_099_312_100_130_925);
+                }
+            }
+            if (blocksLate < 10) {
+                if (blocksLate < 9) {
+                    if (blocksLate == 7) {
+                        return _scaleReward(beans, 2_306_722_744_040_364_517);
+                    }
+                    else { // blocksLate == 8
+                        return _scaleReward(beans, 2_599_272_925_559_383_624);
+                    }
+                }
+                else { // blocksLate == 9
+                    return  _scaleReward(beans, 2_928_925_792_664_665_541); 
+                }
+            }
+            if (blocksLate < 12) {
+                if (blocksLate == 10) {
+                    return _scaleReward(beans, 3_300_386_894_573_665_047);
+                }
+                else { // blocksLate == 11
+                    return _scaleReward(beans, 3_718_958_561_925_128_091); 
+                }
+            }
+            else { // blocksLate == 12
+               return _scaleReward(beans, 4_190_615_593_600_829_241);
+            }
+        } 
+        if (blocksLate < 19){
+            if (blocksLate < 16) {
+                if (blocksLate < 15) {
+                    if (blocksLate == 13) {
+                        return _scaleReward(beans, 4_722_090_542_530_756_587); 
+                    }
+                    else { // blocksLate == 14
+                        return _scaleReward(beans, 5_320_969_817_873_109_037); 
+                    }
+                }
+                else { // blocksLate == 15
+                    return _scaleReward(beans, 5_995_801_975_356_167_528); 
+                }
+            }
+            if (blocksLate < 18) {
+                if (blocksLate == 16) {
+                    return _scaleReward(beans, 6_756_219_741_546_037_047); 
+                }
+                else { // blocksLate == 17
+                    return _scaleReward(beans, 7_613_077_513_845_821_874); 
+                }
+            }
+            return _scaleReward(beans, 8_578_606_298_936_339_361);  // blocksLate == 18
+        }
+        if (blocksLate < 22) {
+            if (blocksLate < 21) {
+                if (blocksLate == 19) {
+                    return _scaleReward(beans, 9_666_588_301_289_245_846); 
+                }
+                else { // blocksLate == 20
+                    return _scaleReward(beans, 10_892_553_653_873_600_447); 
+                }
+            }
+            return _scaleReward(beans, 12_274_002_099_240_216_703);  // blocksLate == 21
+        }
+        if (blocksLate <= 23){ 
+            if (blocksLate == 22) {
+                return _scaleReward(beans, 13_830_652_785_316_216_792); 
+            }
+            else { // blocksLate == 23
+                return _scaleReward(beans, 15_584_725_741_558_756_931); 
+            }
+        }
+        if (blocksLate > 25){ // block rewards are capped at 25 (MAX_BLOCKS_LATE)
```

Comment:
Should this be `>= 25`? Otherwise the 24 and 25 case will both result in `17_561_259_053_330_430_428`